Controller/CRUDController.php

```diff
@@ -456,6 +473,8 @@ public function editAction($id = null, Request $request = null)
                 $templateKey = 'preview';
                 $this->admin->getShow();
             }
+        } else {
+            $session->set($versionKey, $this->admin->getVersion($object));
```

Comment:
@rande Please tell me if you see a way to detect if the form has been submitted from an ``AdminExtension``.

The only way I see is to add a new method in the ``AdminExtension`` class, and call it somewhere after ``$form->handleRequest($request)``.